File: sentimental-mario-less/mario.py
# TODO

def main():
    height = get_height()
    for i in range(height):
        for space in range(height - i - 1):
            print(" ", end="")
        for j in range(i+1):
            print("#", end="")
        print()


def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                return n
        except ValueError:
            print("Not an integer")


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x, space;

    do
    {
        x = get_int("Height: ");
    }
    while (x > 8 || x < 1);

    for (int i = 0; i < x; i++)
    {
        for (space = 0; space < x - i - 1; space++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
         printf("#");
        }

        printf("\n");
    }

    return 0;
}
